typescript/cli/src/config/warp.ts

```diff
@@ -166,6 +165,18 @@ export async function createWarpRouteDeployConfig({
           }),
         };
         break;
+      case TokenType.collateralVault:
+        result[chain] = {
+          mailbox,
+          type,
+          owner,
+          isNft,
+          interchainSecurityModule,
+          token: await input({
+            message: `Enter the erc-4626 vault address on chain ${chain}`,
```

Comment:
nit: we're trying to move away from saying `...on chain ${chain}` because it's kinda awkward for chains that have "chain" in them e.g., "cheesechain"

```suggestion
            message: `Enter the ERC-4626 vault address on ${chain}`,
```

Reply:
I tend include "noun ${var}" so if var is null or undefined it is still clear what should have been there